### ISSUE TITLE
network,getNetworkEvents fail

### ISSUE THREAD

**Issue by cristianoscarpa92** [MitchellGulledge/Meraki-vWAN#23]
the script fail : 
![image](https://user-images.githubusercontent.com/24841413/108998830-c0b79100-76a1-11eb-8a87-e6f0854a9807.png)

I try to manually request the same API by adding ?productType=appliance and I didn't receive error

The script also create object on Azure with secondary MX public ip
![image](https://user-images.githubusercontent.com/24841413/108998022-9dd8ad00-76a0-11eb-8f29-b406732ed30e.png)
![image](https://user-images.githubusercontent.com/24841413/108998199-d4162c80-76a0-11eb-84c1-0c8743c26173.png)

**Comment by MitchellGulledge** [comment]
Hey cristianoscarpa92 can you share your Meraki dashboard mac address so I can have a look? Additionally is this an appliance network or a combined network? Apologies for delay.

**Comment by MitchellGulledge** [comment]
When looking at the organization I do see that all your networks are in combined networks and the network ID in the output you pasted above has L_ indicating other things. Let me make sure the code isnt only built to failover appliance networks. Will update once I have results.

**Comment by MitchellGulledge** [comment]
Good news is I was able to replicate the issue, will have update and fix out later today.

```
2021-03-11 10:49:57       meraki:    ERROR > networks, getNetworkEvents - 400 Bad Request, {'errors': ['productType is required']}
Traceback (most recent call last):
  File "vwan-fail.py", line 14, in <module>
    events_response = MerakiConfig.sdk_auth.networks.getNetworkEvents(
  File "/usr/local/lib/python3.8/site-packages/meraki/api/networks.py", line 551, in getNetworkEvents
    return self._session.get_pages(metadata, resource, params, total_pages, direction, event_log_end_time)
  File "/usr/local/lib/python3.8/site-packages/meraki/rest_session.py", line 283, in get_pages
    response = self.request(metadata, 'GET', url, params=params)
  File "/usr/local/lib/python3.8/site-packages/meraki/rest_session.py", line 262, in request
    raise APIError(metadata, response)
meraki.exceptions.APIError: networks, getNetworkEvents - 400 Bad Request, {'errors': ['productType is required']}
```

I am able to replicate the error when the network ID begins with L_. Easy fix/validation :) Again apologies for the delay

**Comment by MitchellGulledge** [comment]
``` python
# retrieving third party vpn events for network that is detected as down
events_response = MerakiConfig.sdk_auth.networks.getNetworkEvents(
    'xxxxxxxxxxxx', # inputting network ID but vpns['networkId'] is the value in the code
    total_pages=1,
    includedEventTypes = 'vpn',
    productType= 'appliance'  # here is the fix for the issue you are seeing :) Will add and repack this should take care of everything
)

print(events_response)
```

**Comment by MitchellGulledge** [comment]
Fix is merged :) Can you try redeploying? @cristianoscarpa92

**Comment by cristianoscarpa92** [comment]
Now the are no errors on the script, but I have a new issue.
Azure is configured with Appliance IP address instead of Shared IP.
I did packet capture on Meraki switch port and the tunnel is initiated by Shared IP and not by MX IP.
![image](https://user-images.githubusercontent.com/24841413/110939166-47709d00-8335-11eb-90c6-16bb1cf938b7.png)
![image](https://user-images.githubusercontent.com/24841413/110939265-6ec76a00-8335-11eb-8730-aa645888b81e.png)

As workaround I removed the use of virtual IP
